Framework/PythonInterface/test/python/plugins/algorithms/IndirectTransmissionTest.py

```diff
@@ -123,12 +123,11 @@ def test_mass_density(self):
         ws = IndirectTransmission(Instrument=instrument, Analyser=analyser, Reflection=reflection,
                                   ChemicalFormula=formula, DensityType='Mass Density', Density=density, Thickness=thickness)
 
-        ref_result = [6.65800, 0.82122, 2.58186, 53.50693, 56.08880, 0.03342, 0.1, 0.82676, 0.17096]
+        ref_result = [6.65800, 0.82122, 2.58186, 53.50693, 56.08880, 0.100272, 0.1, 0.56512, 0.43021]
         mass_values = ws.column(1)
         np.testing.assert_array_almost_equal(mass_values, ref_result, decimal=4)
 
-        # mass density 1.0 = number density 0.033424 for water
-        density = 0.033424
+        density = 0.100272
```

Comment:
Same question here. I don't quite understand if we are still using the `Number Density` below why that needs to change as the `MaterialBuilder` was only corrected for the mass density setting wasn't it?